FlowCrypt/Controllers/SetupImap/SetupImapViewController.swift

```diff
@@ -529,8 +532,8 @@ extension SetupImapViewController {
         globalRouter.signIn(with: .other(.session(user)))
     }
 
-    private func checkCurrentUser() -> Result<UserRealmObject, UserError> {
-        guard user != UserRealmObject.empty, user.email != UserRealmObject.empty.email else {
+    private func checkCurrentUser() -> Result<User, UserError> {
+        guard user != User.empty, user.email != User.empty.email else {
             return .failure(.empty)
         }
```

Comment:
In the future all similar `Result` usages should get rewritten to throw instead, if the goal is to communicate success / failure. 

Besides, current user should always come through method parameters, as per #863 . 

Filing an issue for the `Result` refactor.